libs/sdk-ui-ext/src/internal/components/pluggableVisualizations/headline/PluggableHeadline.tsx

```diff
@@ -104,6 +105,7 @@ export class PluggableHeadline extends AbstractPluggableVisualization {
         if (props.featureFlags?.enableNewHeadline) {
             this.initializeProperties(props.visualizationProperties);
         }
+        this.keepPrimaryDerivedMeasureOnly = false;
```

Comment:
I would remove this line and just initialize the value when you define the member, i.e., `private keepPrimaryDerivedMeasureOnly = false;`